### ISSUE TITLE
[BUG] Incorrect documentation

### ISSUE THREAD

**Issue by codebrain** [marqo-ai/marqo#340]
Documentation at: https://docs.marqo.ai/0.0.13/API-Reference/search/

![image](https://user-images.githubusercontent.com/148974/220060107-13d3bc2c-5da1-4490-98e8-88c2ea196b18.png)
...mentions this is a `string`, but the model declares an array.

https://github.com/marqo-ai/marqo/blob/085815e802ee633935bf261e05fd4a72740bc42e/src/marqo/tensor_search/models/api_models.py#L23

(also, there is a typo on the page: `Dicitionary`)

**Comment by codebrain** [comment]
Documentation at: https://docs.marqo.ai/0.0.13/API-Reference/search/

![image](https://user-images.githubusercontent.com/148974/220222670-3352db67-2acb-452c-842a-1616431fc519.png)

`highlights` is shown as an object/dictionary, but the API returns an array:

```json
{
  "hits": [
    {
      "Description": "The EMU is a spacesuit that provides environmental protection",
      "Title": "Extravehicular Mobility Unit (EMU)",
      "_id": "article_591",
      "_score": 1.9782796,
      "_highlights": []
    },
    {
      "Description": "A 13th-century travelogue describing Polo's travels",
      "Title": "The Travels of Marco Polo",
      "_id": "f53d3812-8c92-4031-80e3-b7e9e49f5812",
      "_score": 0,
      "_highlights": []
    }
  ],
  "query": "What is the best outfit to wear on the moon?",
  "limit": 20,
  "offset": 0,
  "processingTimeMs": 32
}
```

**Comment by codebrain** [comment]
Documentation at: https://docs.marqo.ai/0.0.13/API-Reference/search/

![image](https://user-images.githubusercontent.com/148974/220223167-2d14c591-f9a2-4da0-9e1b-fa652f3aca2b.png)

Indicates the highlighting parameter is named `highlight`, but the server and client use `showHighlights`

https://github.com/marqo-ai/py-marqo/blob/8242276b8353f496d2640abc11db498ac8b4d662/src/marqo/index.py#L167

https://github.com/marqo-ai/marqo/blob/5dcf4f7de7923fb794d4f69b789e7c24b28ae6b5/src/marqo/tensor_search/api.py#L280

**Comment by pandu-k** [comment]
Hi @codebrain - thanks for pointing these issues out! I've fixed the documentation. The only remaining issue is the inconsistent highlights type. It actually is a dictionary when the search is a tensor-type, it only changes (incorrectly) to an array when search_method is set to lexical (where it will always be an empty array). This issue has been around long enough that fixing it might constitute a breaking change. This bug has been added to the list of breaking changes scheduled for our next major release.
